### DISCUSSION TITLE
Quick way to truncate column content?

### DISCUSSION THREAD

**Discussion by halloleo** [saulpw/visidata#2229] [Q&A] [answered]
I found a way to truncate column content: Say, you want to truncate the current column to 10 chars. For this, do:

 `gs`, `g*` and type:`^(.{10}).*/\1`

It works, but is there a more direct method to do so?

**Comment by anjakefala** [comment, marked as answer]
`gs` `g=` and then type `ColumnName[:10]` seems to also do the trick!

**Reply by halloleo** [reply]
That's a good one too.

It is important to know though, that this method is a "copy": **You need to be in the correct column** (as with the regex method) _and_ you also need to type that column name in the select expression.

So, if you want to truncate the column `WaterConsumption`, your cursor needs to be in the `WaterConsumption` column  and as select expression you type `WaterConsumption[:10]`.

**Reply by saulpw** [reply]
You can also use `curcol` for the current column, instead of typing the actual column name.

**Reply by halloleo** [reply]
So `curcol[:10]` would work as the select expression, right? That's cool!
